tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -268,6 +269,8 @@ def get_mimetext_message(self, sqs_message, resources, to_addrs):
         message = MIMEText(body, email_format)
         message['From'] = from_addr
         message['To'] = ', '.join(to_addrs)
+        if sqs_message['action'].get('cc'):
+            message['Cc'] = ', '.join(sqs_message['action']['cc'])
```

Comment:
are these the resolved emails? cc can also include resource-owner or event-owner so you'll want to make sure to resolve the emails first before proceeding